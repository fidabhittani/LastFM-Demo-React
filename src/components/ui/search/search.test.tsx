import React from "react";
import { shallow } from "enzyme";
import SearchBar from "./index";

/** SearchBar TEST */
describe("SearchBar Root Component Tests", () => {
  it("SearchBar Component renders correctly", () => {
    const resultWrapper = shallow(
      <SearchBar getSearch={(search: string) => console.log("Searching...")} />
    );
    expect(resultWrapper).toMatchSnapshot();
  });
});
