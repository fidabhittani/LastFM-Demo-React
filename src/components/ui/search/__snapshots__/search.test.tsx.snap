// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar Root Component Tests SearchBar Component renders correctly 1`] = `ShallowWrapper {}`;
